packages/p2p/src/components/advertiser-page/advertiser-page.scss

```diff
@@ -64,7 +64,6 @@
     }
 
     &__page-return-header {
-        display: flex;
         position: relative;
```

Comment:
`display: flex` causes the header to be cropped out due to overlay:
![Screenshot 2022-09-19 at 2 54 03 PM](https://user-images.githubusercontent.com/103016120/190964643-1cfb1edd-00ff-4b29-bbdd-16d902d5b104.png)
![Screenshot 2022-09-19 at 2 54 06 PM](https://user-images.githubusercontent.com/103016120/190964657-f7113637-c5de-4335-8471-77efe4111c0c.png)

Reply:
NOTE: Only `advertiser-page.jsx` is using this class